tests/Integration/Schema/Directives/CreateDirectiveTest.php

```diff
@@ -77,7 +79,8 @@ public function testCanCreateFromInputObject(): void
 
     public function testCreatesAnEntryWithDatabaseDefaultsAndReturnsItImmediately(): void
     {
-        $this->schema .= /** @lang GraphQL */ '
+        $this->schema .= /** @lang GraphQL */
+            '
```

Comment:
```suggestion
        $this->schema .= /** @lang GraphQL */ '
```

Same with the other changes like this.